source/_integrations/bosch_alarm.markdown

```diff
@@ -34,12 +36,18 @@ The **Bosch Alarm Panel** {% term integration %} allows you to connect your [Bos
 The following {% term entities %} are provided:
 
 - [Alarm Control Panel](#alarm-control-panel)
+- [Sensor](#sensor)
 
 ### Alarm Control Panel
 
 This integration adds an Alarm Control Panel device for each configured area, with the ability to issue arm/disarm commands.
 This entity reports state (_disarmed_, _armed_away_, etc.).
 
+## Sensor
+
+A sensor is provided for the panel containing the current faults from it.
```

Comment:
```suggestion
A sensor is provided for the panel to show the current error messages (faults).
```
Please correct me if I got this wrong. Added error message because it seems a more commonly used term to me.